### ISSUE TITLE
Duplicate org.json in dependencies

### ISSUE THREAD

**Issue by hansjakobli** [MuleChain-Project/mulechain-ai-connector#7]
Using v0.1.0, the org.json modules are mentioned twice as dependencies.

No 1:
```
		<dependency>
			<groupId>org.json</groupId>
			<artifactId>json</artifactId>
			<version>20240303</version>
		</dependency>
```

No 2:
```
		<dependency>
			<groupId>org.json</groupId>
			<artifactId>json</artifactId>
			<version>20210307</version>
		</dependency>
```

Leading to the following warning:
```
[WARNING] Failed to build parent project for com.mule.mulechain:mulechain-ai-connector:mule-extension:0.1.0
[WARNING]
[WARNING] Some problems were encountered while building the effective model for com.mule.mulechain:mulechain-ai-connector:mule-extension:0.1.0
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.json:json:jar -> version 20240303 vs 20210307 @ line 124, column 15
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]```

**Comment by amirkhan-ak-sf** [comment]
@hansjakobli thanks for reporting. We will fix it first in the develop branch and later with a new release on the master branch.

**Comment by amirkhan-ak-sf** [comment]
Fixed with /[develop 0.1.27](https://github.com/MuleChain-Project/mulechain-ai-connector/commit/4a2e833a6d7f44740fcfb395ee17941bc6450575)
